controllers/PollController.js

```diff
@@ -0,0 +1,81 @@
+var flash = require('../utilities/flash.js').flash;
+var Model = require('../models/models.js');
+
+module.exports = {
+ 
+ deletePoll: function(req, res){
+  var id= req.query.id;
+  Model.PollsModel.remove({_id:id}, function(err, result){
+   if (err) {
+    flash(req, 'danger', 'poll was not deleted', 'try again');
+    console.log ("I could not delete the poll!")}
+   else{
+    flash(req, 'success', 'poll deleted', 'your poll was deleted');
+    res.redirect(302,'/polls/myPolls'); 
```

Comment:
you don't need to specify the http status for every redirect... you can use just redirect(path)